src/EdFi.AnalyticsMiddleTier.DataStandard32/Engage/MSSQL/0006-Upcoming-Submissions-Map.sql

```diff
@@ -0,0 +1,45 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+WITH UpcomingSubmission as (
+	SELECT 
+		DescriptorConstant.DescriptorConstantId,
+		d.DescriptorId
+	FROM 
+		analytics_config.DescriptorConstant
+	CROSS JOIN (
+		SELECT 
+			Descriptor.DescriptorId
+		FROM
+			edfi.Descriptor
+		INNER JOIN
+			lmsx.AssignmentCategoryDescriptor ON
```

Comment:
Shouldn't this be the `SubmissionStatusDescriptor` table?

Reply:
Fixed, thanks!